### PR TITLE
Added a list of definite/indefinite articles and prepositions to not …

### PR DESCRIPTION
**Pull request by skeltonath**
…replace in move handler

### CODE REVIEW

**Comment by skeltonath** [comment]
This fixes issue #8

**Comment by skeltonath** [comment]
also fixes issue #7

File: handlers/movie.js [hunk 1]
```diff
@@ -80,8 +80,8 @@ async function movie(channel, message, params) {
         return;
       }
 
-      channel.send(format('%s [%s]', replaceRandomWords(movie.Title, params, 1), id));
-      channel.send(replaceRandomWords(movie.Plot, params));
+      channel.send(replaceRandomWords(movie.Title, params, 1));
```
Comment (thekanbo) [inline handlers/movie.js]: Title should be sent in title case

File: handlers/movie.js [hunk 2]
```diff
@@ -80,8 +80,8 @@ async function movie(channel, message, params) {
...
+      channel.send(replaceRandomWords(movie.Plot, _.capitalize(params)));
```
Comment (thekanbo) [inline handlers/movie.js]: Probably don't need case changed in plot text

Comment (skeltonath) [reply handlers/movie.js]: oops, did it backwards

**Pull request merged**